apis/core-api/src/api/routes/v1/matches/comments/ControllerAdapter.ts

```diff
@@ -0,0 +1,88 @@
+import MatchCommentController from 'core/controllers/MatchCommentController';
+
+import { Context } from 'types';
+
+class ControllerAdapter {
+  static async create(ctx: Context) {
+    /** VALIDATIONS, PARAMETERS */
+    const { id: matchId } = ctx.request.params;
+    const { content } = ctx.request.body;
```

Comment:
Is this content supports emojis?

Reply:
that was depends from database charset and collate, we are using UTF8-MB4 (which means, yes, emojis was supported)